src/InputField.js

```diff
@@ -40,16 +40,16 @@ class InputField extends React.Component {
   constructor(props) {
     super(props)
 
-    let hasDefaultValue
+    let hasInitialValue
 
     React.Children.forEach(props.children, child => {
       if (child && child.type === Input) {
-        hasDefaultValue = child.props.defaultValue
+        hasInitialValue = child.props.value
```

Comment:
Should this be `!!child.props.value` to set the state as a boolean?

Reply:
yeah definitely. I've updated it.